BioChem/UsersGuide/CopyRight.mo

```diff
@@ -0,0 +1,7 @@
+within BioChem.UsersGuide;
+
+class CopyRight
+  extends Modelica.Icons.Information;
+  annotation(
+    Documentation(info = "<html><head></head><body>Copyright © 2005-2015 MathCore Engineering AB, Linköpings universitet and Modelica Association</body></html>"));
```

Comment:
You could probably bump that to 2021 and add your own name

Reply:
Done and have added your name as well under ReleaseNotes